pachyderm/values.yaml

```diff
@@ -24,14 +24,14 @@ dash:
     # tag is the image repo to pull from; together with repository it
     # replicates the --dash-image argument to pachctl deploy.
     tag: "0.5.57"
-  # resources specifies the resource request
-  resources: #Can't zero out resources with {}
-    limits:
-      cpu: "1"
-      memory: "2G"
-    requests:
-      cpu: "1"
-      memory: "2G"
+  # resources specifies the resource request and limits
```

Comment:
Minor formatting nit: I think this should be ‘resources specifies the resource request*s* and limits*.*’ in order to match the other comments. Ditto below.